lib/dry/system/loader.rb

```diff
@@ -61,7 +63,10 @@ def call(component, *args)
         # @api public
         def constant(component)
           inflector = component.inflector
-          inflector.constantize(inflector.camelize(component.const_path))
+          const_name = inflector.camelize(component.const_path)
```

Comment:
Oops, this is leftover from some other changes I had. Happy to get rid of it, but I find it does make the code a little more readable

Reply:
Yeah, this is fine by me :)